lib/async_hooks.js

```diff
@@ -125,11 +125,6 @@ function createHook(fns) {
 }
 
 
-function executionAsyncId() {
-  return async_id_fields[kExecutionAsyncId];
-}
-
-
```

Comment:
nit: `executionAsyncId` and `triggerAsyncId` should probably stay together. Either both move or neither.

Reply:
Works for me. I'll move both. (I wanted to avoid requiring both files during bootstrap.)